plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -33,6 +34,14 @@ private DynamicTablePqlExtractor()
     public static String extractPql(DynamicTable table, TupleDomain<ColumnHandle> tupleDomain)
     {
         StringBuilder builder = new StringBuilder();
+        Map<String, String> queryOptions = table.getQueryOptions();
+        queryOptions.keySet().stream().sorted().forEach(
```

Comment:
Why do we need to sort? Why not use `queryOptions.forEach((key, value) -> ...` ?

Reply:
This is to make the generated set key-value pair statements ordering fixed.
The ordering itself doesn't matter.